src/core/inductive.ml

```diff
@@ -0,0 +1,9 @@
+open Timed
+open Terms
+
+type inductive =
+  { ind_constructors : sym list   ; (** the list of constructors           *)
+    ind_prop         : sym option ; (** one inductive principle (Prop one) *) }
```

Comment:
No option type here I think. If we cannot generate the induction principle then the `inductive` command is useless and should fail, don't you think?

Reply:
Yes, I think too!